docs/guides/code-splitting.md

```diff
@@ -0,0 +1,112 @@
+# Code Splitting
+
+## Basic principles
+
+In a large application, there is a need to break the code into many small bundles. This allows you to make the project come alive much earlier so that your users do not feel uncomfortable. See more info about code splitting in [webpack docs](https://webpack.js.org/guides/code-splitting/)
+
+We also adhere to this idea and tried to make Reatom with the expectation of this behavior.
```

Comment:
maybe => "Reatom supports code splitting through Webpack. Please refer to the example below."

Reply:
Code splitting doesn't have to go through webpack. It can be manual. The main message is that it is not necessary to ship the entire application at once. Reatom is able to work with this approach normally.